package.json

```diff
@@ -6,7 +6,8 @@
   "module": "dist/index.es.js",
   "browser": "dist/index.cjs.js",
   "files": [
-    "dist"
+    "dist",
+    "index.d.ts"
```

Comment:
I'm still getting familiar with ts. Should this just go in dist? Do we need to add a "types" section in package.json?

Reply:
I've added in `typings`, thanks for the highlight.